ush/rocoto/workflow_utils.py

```diff
@@ -201,8 +204,11 @@ def create_wf_task(task, cdump='gdas', cycledef=None, envar=None, dependency=Non
     # Add PARTITION_BATCH to all non-service jobs on Orion (SLURM)
     if get_scheduler(detectMachine()) in ['slurm'] and detectMachine() in ['ORION']:
         task_dict['partition'] = '&PARTITION_BATCH;'
-    # Add PARTITION_SERVICE to all service jobs (SLURM)
-    if get_scheduler(detectMachine()) in ['slurm'] and task in ['getic','arch','earc']:
+    # Add PARTITION_SERVICE to all service jobs on Hera (SLURM)
+    if get_scheduler(detectMachine()) in ['slurm'] and task in ['getic','arch','earc'] and detectMachine() in ['HERA']:
```

Comment:
You don't need the `and detectMachine() in ['HERA']` part here.

Reply:
I did some modify to workflow_utils.py.